### ISSUE TITLE
Do not clone the input values

### ISSUE THREAD

**Issue by sushant12** [feliperenan/dynamic_nested#7]
I think we should just clone the input fields and not the values.

For eg:

If i have an email field and I enter my email addr and add a new email field, it should create a new empty email field.

If you like the idea, I have worked on the code in my separate branch and I can send you a PR

**Comment by feliperenan** [comment]
Hi @sushant12 

In the project that I haven been using it I need to have values copied as well. What about if we provide some option for that, so that the decision would be made when building the nested form 🤔 .

What do you think?

**Comment by sushant12** [comment]
@feliperenan We can use the hook methods https://github.com/feliperenan/dynamic_nested/blob/master/js/dynamic_nested.js#L133  to copy over the input values. 

What do you say??

**Comment by feliperenan** [comment]
That's true, we could use `afterAdd` callback for that. But they will need to remember to handle select fields with something like that in their application:

https://github.com/feliperenan/dynamic_nested/blob/627807ee81cb0d8c22f98aa5d8891061bd0d5222/js/dynamic_nested.js#L107

Besides that, cloning values is the default behaviour when cloning elements. It seems that only execption are select fields. That's why I'd like to keep this behavior as default.

When I suggested a option for that, I thought in something like that:

```JS
// dynamic_nested.js
add($allNested) {
  if (option.copyInputValues) {
    // execute script to copy select values.
  }

  if (!option.copyInputValues) {
    // clean copied input values since they are copied by default.
  }
}
```

Then developers who don't want to clone with value copied would do something like this when initializing it.
```JS
new DynamicNested(copyInputValues: false)
```

That's would satisfy both scenario. But honestly, this library doesn't have many users yet, so we could try your suggestion as well. So fell free to open a pull-request for that :).
